### DISCUSSION TITLE
Does Next-on-pages support Turborepo?

### DISCUSSION THREAD

**Discussion by iicdii** [cloudflare/next-on-pages#304] [Q&A]
I couldn't find a description of the monorepo tool (e.g., Turborepo) in the document. I'm wondering if you're supporting or have a plan to support monorepo. If it supports, It would be nice to have an example for it.

**Comment by dario-piotrowicz** [comment]
Hi @iicdii, sorry but I am not sure I understand the question, could you explain what kind of support are you referring to/thinking of?

**Reply by iicdii** [reply]
> Monorepos or repositories with multiple codebases/applications currently cannot use the automatic GitHub/GitLab integration to build multiple sites from the same repository. ([Known issues](https://developers.cloudflare.com/pages/platform/known-issues/#builds-and-deployment))

Ok so I just found it in the documentation. As the document says, I need an example of how to deploy multiple sites from a single repository.

**Reply by dario-piotrowicz** [reply]
you can't currently do that via the Cloudflare Pages GitHub integration, the only way to do it (besides manually deploying the applications that is) is by using the [Pages Github Actions](https://github.com/cloudflare/pages-action)
